# This fix comes from the upstream Pillow fork of PIL:
# https://github.com/python-pillow/Pillow/commit/c6040f618d8f2706a7b46d1cdf37d1a587f9701f

--- Imaging-1.1.7-orig/_imagingft.c	2009-10-31 17:44:12.000000000 -0700
+++ Imaging-1.1.7/_imagingft.c	2015-04-09 08:30:10.889971000 -0700
@@ -70,7 +70,11 @@
     const char* message;
 } ft_errors[] =
 
+#if defined(USE_FREETYPE_2_1)
+#include FT_ERRORS_H
+#else
 #include <freetype/fterrors.h>
+#endif
 
 /* -------------------------------------------------------------------- */
 /* font objects */
